### PR TITLE
update pipeline result

### PR DESCRIPTION
**Pull request by chengjoey**
#### What type of this PR

Add one of the following kinds:
/kind bugfix

#### What this PR does / why we need it:
update pipeline result

#### Which issue(s) this PR fixes:

- Fixes #your-issue_number
- [Erda Cloud Issue Link](paste your link here)

#### Specified Reviewers:

/assign @your-reviewer

#### Need cherry-pick to release versions?

Add comment like `/cherry-pick release/1.0` when this PR is merged.

> For details on the cherry pick process, see the [cherry pick requests](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md#how-to-cherry-pick-a-merged-pr) section under [CONTRIBUTING.md](https://github.com/erda-project/erda/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by chengjoey** [comment]
image
image

File: modules/pipeline/dbclient/op_pipeline_task.go [hunk 1]
```diff
@@ -267,8 +267,8 @@ func (client *Client) CleanPipelineTaskResult(id uint64, ops ...SessionOption) e
 	session := client.NewSession(ops...)
 	defer session.Close()
 
-	if _, err := session.Table("pipeline_tasks").ID(id).
-		Update(map[string]interface{}{"result": nil}); err != nil {
+	if _, err := session.Table("pipeline_tasks").
```
Comment (sfwn) [inline modules/pipeline/dbclient/op_pipeline_task.go:270]: judge id is 0

**Review by sfwn — CHANGES_REQUESTED** [review]

File: modules/pipeline/dbclient/op_pipeline_task.go (hunk 1 above)
Comment (chengjoey) [reply modules/pipeline/dbclient/op_pipeline_task.go:270]: don't need to judge zero, ID()-func need struct, map is invalid

Comment (sfwn) [reply modules/pipeline/dbclient/op_pipeline_task.go:270]: Defensive programming is needed. Someone maybe make a mistake here in the future.

**Review by sfwn — CHANGES_REQUESTED** [review]

**Review by sfwn — APPROVED** [review]

**Pull request merged**